libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -145,6 +149,51 @@ void RCOutput::init()
     _initialised = true;
 }
 
+// start the led thread
+bool RCOutput::start_led_thread(void)
+{
+#ifdef HAL_NO_LED_THREAD
+    return false;
+#else
+    WITH_SEMAPHORE(led_thread_sem);
+
+    if (led_thread_ctx) {
+        return true;
+    }
+
+    if (!hal.scheduler->thread_create(FUNCTOR_BIND_MEMBER(&RCOutput::led_thread, void), "led", LED_THD_WA_SIZE, AP_HAL::Scheduler::PRIORITY_LED, 0)) {
```

Comment:
if we have two groups with LEDs we could end up with two threads, as there is a race in the settings of led_thread_ctx